qiskit_nature/problems/sampling/protein_folding/__init__.py

```diff
@@ -0,0 +1,86 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Protein Folding Problems (:mod:`qiskit_nature.problems.sampling.protein_folding`)
+=================================================================================
+
+The protein to be folded is defined in a Peptide class. Each peptide consists of one and only one
+main chain and optionally several side chains. Side chains cannot be attached to first, second or
+last main bead which is an assumption of the algorithm without loss of generality (see the paper
+cited below). Each chain consists of beads that encode information about the turn that follows
+to another main bead (in case of main beads) or into a side bead (in case of side beads).
```

Comment:
`to another` does not work. What do you want to say here?

Reply:
_A chain consists of beads that contain information about their relative position to a next main bead (in case of main beads) or to a previous main bead (in case of a first side bead) or to a previous side bead (in case of other side beads)._
Does it work better?

Reply:
Hmm.. Maybe we can do that case distinction a little clearer:
```
A chain consists of beads containing information about their relative position to other beads:
* main beads reference previous main beads
* _first_ side beads reference the (branching) main bead
* other side beads reference previous side beads
```
This may still be improved but I think a list may be clearer than an excessively long sentence.

Reply:
Thanks